examples/divider/pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["briefcase"]
+
+[tool.briefcase]
+project_name = "Divider Demo"
+bundle = "org.beeware"
+version = "0.0.1"
+url = "https://beeware.org"
+license = "BSD license"
+author = 'Tiberius Yak'
+author_email = "tiberius@beeware.org"
+
+[tool.briefcase.app.dialogs]
+formal_name = "Dialog Demo"
+description = "A testing app"
+sources = ['dialogs']
```

Comment:
typo - this should be `divider`

(as should the other references to dialogs)

Reply:
fixed